packages/app-cli/tests/services_SearchEngine.js

```diff
@@ -515,4 +515,14 @@ describe('services_SearchEngine', function() {
 		expect((await engine.search('"- [ ]"', { searchType: SearchEngine.SEARCH_TYPE_BASIC })).length).toBe(1);
 		expect((await engine.search('"[ ]"', { searchType: SearchEngine.SEARCH_TYPE_BASIC })).length).toBe(2);
 	}));
+
+	it('should not mistake cyrillic "l" for latin "n"', asyncTest(async () => {
+		const n1 = await Note.save({ title: 'latin n', body: 'n' });
+		const n2 = await Note.save({ title: 'cyrillic l', body: 'л' });
+
+		await engine.syncTables();
+
+		expect((await engine.search('n')).length).toBe(1);
+		expect((await engine.search('л')).length).toBe(1);
```

Comment:
Could you check the result content rather than just the number of search results please? For example with this test if the search engine suddenly starts returning "latin n" for both queries, we won't know about it.

Reply:
Fixed to check ids, getting the content requires an extra step as search does not return it.